docs/source/development/index.rst

```diff
@@ -59,8 +59,8 @@ releases.
 
 .. code::
 
-    $ python setup.py sdist --formats=zip,gztar
-    $ python setup.py bdist_wheel
+    python setup.py sdist --formats=zip,gztar
+    python setup.py bdist_wheel --universal
```

Comment:
We can't set this in IPython if we want to continue to provide the `ipython2` and `ipython3` executables.